auto_parking/scripts/parking.py

```diff
@@ -60,8 +61,7 @@ def process_scan(self, m):
                 rospy.signal_shutdown("Done parking.")
             elif self.dist2Neato < 0.3:
                 print "Neato was too close to park!"
-            
-          
+               
```

Comment:
Yes! Maybe we should do the same if the spot is too small or if we never get a time 1 (maybe if too much time has passed and we haven't gotten one?)